README.md

```diff
@@ -86,580 +103,13 @@ Note: The resulting "splunk-kafka-connect*.tar.gz" package is self-contained. Bu
-
-This section documents how to configure Kafka Connect if your Kafka Cluster is secured using [SASL/PLAIN](http://kafka.apache.org/documentation/#security_sasl_plain).
-
-Configure the Kafka Connect worker and consumer settings to use SASL/PLAIN in **config/connect-distributed.properties**
-
-```
-# Worker security are located at the top level
-security.protocol=SASL_SSL
-sasl.mechanism=PLAIN
-
-# Sink security settings are prefixed with "consumer."
-consumer.security.protocol=SASL_SSL
-consumer.sasl.mechanism=PLAIN
-```
-
-Modify **bin/connect-distributed.sh** by editing the `EXTRA_ARGS` environment variable. Pass in the location of the JAAS conf file.
-
-```
-EXTRA_ARGS=${EXTRA_ARGS-'-name connectDistributed -Djava.security.auth.login.config=/root/kafka_connect_jaas.conf'}
-```
-
-See [Confluent's documentation](https://docs.confluent.io/current/kafka/sasl.html#sasl-configuration-for-kafka-clients) for more information on configuring Kafka Connect using JAAS.
-
-For example, a Kafka Client JAAS file for SASL/PLAIN.
-
-```
-KafkaClient {
-  org.apache.kafka.common.security.plain.PlainLoginModule required
-  username="alice"
-  password="alice-secret";
-};
-```
-
-Start Kafka Connect
-
-```
-./bin/connect-distributed.sh config/connect-distributed.properties
-```
-
-Workers and sink tasks should work with your SASL/PLAIN secured cluster.
-
-### SASL/SCRAM-SHA-256 and SASL/SCRAM-SHA-512
-
-This section documents how to configure Kafka Connect if your Kafka Cluster is secured using [SASL/SCRAM](http://kafka.apache.org/documentation/#security_sasl_scram).
-
-Configure the Kafka Connect worker and consumer settings to use SASL/SCRAM in **config/connect-distributed.properties**
-
-```
-# Worker security are located at the top level
-security.protocol=SASL_SSL
-sasl.mechanism=SCRAM-SHA-256 (or SCRAM-SHA-512)
-
-# Sink security settings are prefixed with "consumer."
-consumer.security.protocol=SASL_SSL
-consumer.sasl.mechanism=SCRAM-SHA-256 (or SCRAM-SHA-512)
-```
-
-Modify **bin/connect-distributed.sh** by editing the `EXTRA_ARGS` environment variable. Pass in the location of the JAAS conf file.
-
-```
-EXTRA_ARGS=${EXTRA_ARGS-'-name connectDistributed -Djava.security.auth.login.config=/root/kafka_connect_jaas.conf'}
-```
-
-See [Confluent's documentation](https://docs.confluent.io/current/kafka/sasl.html#sasl-configuration-for-kafka-clients) for more information on configuring Kafka Connect using JAAS.
-
-For example, a Kafka Client JAAS file for SASL/SCRAM.
-
-```
-KafkaClient {
-  org.apache.kafka.common.security.scram.ScramLoginModule required
-  username="alice"
-  password="alice-secret";
-};
-```
-
-Start Kafka Connect
-
-```
-./bin/connect-distributed.sh config/connect-distributed.properties
-```
-
-Workers and sink tasks should work with your SASL/SCRAM secured cluster.
-
-## Configuration
```

Comment:
Keep the configuration section in README with all the parameters

Reply:
Done